templates/releases/activity.html

```diff
@@ -9,51 +9,63 @@
 
 {%- block body -%}
     <div class="container">
-        <div id="releases-activity-chart"></div>
+        <canvas id="releases-activity-chart"></canvas>
     </div>
 {%- endblock body -%}
 
 {# TODO: Do this with tera alone #}
+{%- block css -%}
+    <link rel="stylesheet" href="/-/static/chartjs/chart.min.css">
+{%- endblock -%}
 {%- block javascript -%}
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/highcharts/4.2.5/highcharts.js" type="text/javascript"
-        charset="utf-8"></script>
+    <script src="/-/static/chartjs/chart.min.js" type="text/javascript"></script>
 
-    <script type="text/javascript" charset="utf-8">
-        new Highcharts.Chart({
-            chart: {
-                renderTo: 'releases-activity-chart',
-                type: 'line'
-            },
-            title: {
-                text: 'Monthly release activity',
-                x: -20 //center
-            },
-            xAxis: {
-                categories: [
+    <script type="text/javascript">
+        // We're including the CSS file manually to avoid issues with the CSP.
+        Chart.platform.disableCSSInjection = true;
```

Comment:
By default this will load the CSS from the internet?

Reply:
No, without the option it will inject the styling directly into the DOM. [Documentation](https://www.chartjs.org/docs/latest/getting-started/integration.html#content-security-policy)